CMake unable to find jsoncpp.cpp

Every time i open NeoVim i get the message "Done. Restart NeoVim.". In order to find out why it didn't work i decided to build from `~/.local/share/nvim/lazy/nvim-textmate` and got the next error message when running `make`
```
...
config.status: executing libtool commands                                                                                                                              
config.status: executing default commands                                                                                                                              
mkdir -p build                                                                                                                                                         
cd build && cmake ../cmake && make                                                                                                                                     
-- Configuring done (0.0s)                                                         
CMake Error at CMakeLists.txt:82 (add_library):           
  Cannot find source file:

    ../libs/jsoncpp/dist/jsoncpp.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm
  .ccm .cxxm .c++m .h .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90
  .f95 .f03 .hip .
                                                                                   
CMake Error at CMakeLists.txt:82 (add_library):                                    
  No SOURCES given to target: 

CMake Generate step failed.  Build files cannot be regenerated correctly.          
make: *** [Makefile:10: build] Error 1
```

Printing the output from `vim.fn.system({ "make", "build", "-C", target_path })` in `~/.local/share/nvim/lazy/nvim-textmate/lua/nvim-textmate/init.lua` gives the same error message. The error comes from `../cmake/CMakeList.txt` in the

```
add_library( textmate
    SHARED
...
```
section.

--- a/Makefile
+++ b/Makefile
@@ -3,11 +3,12 @@ all: prebuild build install
 .PHONY: prebuild build install
 
 prebuild:
+	cd libs/jsoncpp && ./amalgamate.py
 	cd libs/Onigmo && ./autogen.sh && ./configure
 
 build:
 	mkdir -p build
-	cd build && cmake ../cmake && make
+	cd build && cmake ../cmake && cmake --build .
 	cp build/textmate.so ./lua/nvim-textmate/
 
 install:

